modules/nw-multus-vlan-object.adoc

```diff
@@ -45,18 +45,27 @@ The following object describes the configuration parameters for the VLAN CNI plu
 
 |`dns`
 |`integer`
-|Optional: DNS information to return, for example, a priority-ordered list of DNS nameservers.
+|Optional: DNS information to return. For example, a priority-ordered list of DNS nameservers.
 
 |`linkInContainer`
 |`boolean`
-|Optional: Specifies whether the master interface is in the container network namespace or the main network namespace. Set the value to `true` to request the use of a container namespace master interface.
+|Optional: Specifies whether the `master` interface is in the container network namespace or the main network namespace. Set the value to `true` to request the use of a container namespace `master` interface.
 
 |====
 
+[IMPORTANT]
+====
+Consider the following limitations when you use a VLAN as an additional network for a cluster:
+
+* You cannot specify the same `vlanId` for multiple `NetworkAttachmentDefinition` custom resource definitions (CRDs) that exist in the same project namespace.
```

Comment:
Restriction is in the driver, at the kernel level. Master and VlanId pair should be unique.

You can have multiple nads with the same vlanId but different master interface.